game/scripts/npc/items/item_azazel_tower_watch_4.txt

```diff
@@ -0,0 +1,77 @@
+"DOTAItems"
+{
+    //=================================================================================================================
+    // Recipe: Watch Tower 4
+    //=================================================================================================================
+    "item_recipe_azazel_tower_watch_4"
+    {
+        "ID"                            "8505"
+        // General
+        //-------------------------------------------------------------------------------------------------------------
+        "BaseClass"                     "item_datadriven"
+        "Model"                         "models/props_gameplay/recipe.vmdl"
+        "ItemCost"                      "100"
+        "ItemRecipe"                    "1"
+        "ItemResult"                    "item_azazel_tower_watch_4"
+        "AbilityTextureName"            "custom/recipe/recipe_4"
+        "ItemRequirements"
+        {
+            "01"                        "item_upgrade_core_4"
+        }
+    }
+    //=================================================================================================================
+    // Azazel: Watch Tower 4
+    //=================================================================================================================
+    "item_azazel_tower_watch_4"
+    {
+        "ID"                            "8493"
+        // General
+        //-------------------------------------------------------------------------------------------------------------
+        "BaseClass"                     "item_lua"
+        "ScriptFile"                    "items/azazel_tower_watch.lua"
+        "AbilityTextureName"            "custom/far_sight_4"
+        "AbilityBehavior"               "DOTA_ABILITY_BEHAVIOR_POINT"
+        "AbilityCastRange"              "600"
+        "ItemPermanent"                 "0"
+        "ItemInitialCharges"            "1"
+        "MaxUpgradeLevel"               "4"
+        "ItemBaseLevel"                 "4"
+        "ItemStackable"                 "1"
+        "ItemCost"                      "100"
+        // Special
+        //-------------------------------------------------------------------------------------------------------------
+        "AbilitySpecial"
+        {
+            "01"
+            {
+                "var_type"                  "FIELD_INTEGER"
+                "construction_time"         "10"
+            }
+            "02"
+            {
+                "var_type"                  "FIELD_INTEGER"
+                "bonus_health"              "0 1500 3500 7500"
+            }
+            "03"
+            {
+                "var_type"                  "FIELD_INTEGER"
+                "bonus_vision_range"        "0 500 1000 1500"
```

Comment:
Can be changed to just be the total vision range by using the set vision functions, or the fixed vision modifier properties.